server/client.go

```diff
@@ -224,7 +222,7 @@ func (c *Client) WorldItemAdded(id game.EntityId, worldItem *game.WorldItem) {
 }
 
 func (c *Client) WorldItemUpdated(id game.EntityId, worldItem *game.WorldItem) {
-	c.SendLossy(&MsgEntityState{
+	c.Send(&MsgEntityState{
```

Comment:
Still need to actually measure bandwidth, or at least thow this up on DigitalOcean to try testing it.